packages/vscode-extension/src/webview/views/CreateDeviceView.tsx

```diff
@@ -143,13 +158,15 @@ function CreateDeviceView({ onCreate, onCancel }: CreateDeviceViewProps) {
           <span>System image</span>
           {systemImagesOptions.length === 0 && <span className="codicon codicon-warning warning" />}
         </Label>
-        <div className="form-label" />
         {systemImagesOptions.length > 0 ? (
           <Select
             disabled={!platformSelected}
             className="form-field"
             value={selectedSystemName ?? ""}
-            onChange={(newValue) => selectSystemName(newValue)}
+            onChange={(newValue) => {
+              selectSystemName(newValue);
+              inputRef.current!.value = deviceModel ?? "";
```

Comment:
this is a controlled component, so this shouldn't be here, why have you added this?